typescript/infra/scripts/check/check-owner-ica.ts

```diff
@@ -55,23 +49,26 @@ async function main() {
 
   const mismatchedResults: Record<
     string,
-    { Expected: Address; Actual: Address }
+    { Expected: IcaArtifact; Actual: IcaArtifact }
   > = {};
   for (const chain of checkOwnerIcaChains) {
-    const expectedAddress = icas[chain as keyof typeof icas];
-    if (!expectedAddress) {
+    const expected = awIcas[chain as keyof typeof awIcas];
+    if (!expected) {
       console.error(`No expected address found for ${chain}`);
       continue;
     }
-    const actualAccount = await ica.getAccount(
-      chain,
-      ownerConfig,
-      ownerChainInterchainAccountRouter,
-    );
-    if (!eqAddress(expectedAddress, actualAccount)) {
+    const actualAccount = await ica.getAccount(chain, {
+      ...ownerConfig,
+      ismOverride: expected.ism,
+      routerOverride: ownerChainInterchainAccountRouter,
+    });
+    if (!eqAddress(expected.ica, actualAccount)) {
       mismatchedResults[chain] = {
-        Expected: expectedAddress,
-        Actual: actualAccount,
+        Expected: expected,
+        Actual: {
+          ica: actualAccount,
+          ism: expected.ism,
```

Comment:
Does it make sense to use the expected ism here?

Reply:
mm good point, annoyingly we can't easily recover the ISM address given an ICA address, so I'll just say it's unknown